### PR TITLE
Extractor Extension

### PR DESCRIPTION
**Pull request by doctrino**
## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite/client/data_classes/extractor_extension/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from __future__ import annotations
```
Comment (erlendvollset) [inline cognite/client/data_classes/extractor_extension/__init__.py:1]: Let's name this package extractor_extensions (s at the end) to mirror the space name.

**Comment by erlendvollset** [comment]
Very nice that we can generate these btw 💯

File: cognite/client/data_classes/extractor_extension/__init__.py (hunk 1 above)
Comment (doctrino) [reply cognite/client/data_classes/extractor_extension/__init__.py]: Nice catch! Fixed

**Comment by doctrino** [comment]
> Can we put the cdm/extractor_extensions under data_classes/data_modeling instead of data_classes?



> Can we put the cdm/extractor_extensions under data_classes/data_modeling instead of data_classes?

Done :)

**Comment by erlendvollset** [comment]
Let's also move the `cdm` package.

File: cognite/__init__.py
[the remark this answers is not in the document]
Comment (doctrino) [reply cognite/__init__.py:1]: Luckily we have a test that catches it. Already fixed.

[the remark this answers is not in the document]
Comment (haakonvt) [reply cognite/__init__.py:1]: You're welcome 🤓

**Pull request merged**